internal/appsec/remoteconfig_test.go

```diff
@@ -119,102 +119,154 @@ func TestMergeRulesData(t *testing.T) {
 	for _, tc := range []struct {
 		name     string
 		update   remoteconfig.ProductUpdate
-		expected []config.RuleDataEntry
+		expected config.RulesFragment
 		statuses map[string]rc.ApplyStatus
 	}{
 		{
 			name:     "empty-rule-data",
 			update:   map[string][]byte{},
-			statuses: map[string]rc.ApplyStatus{"some/path": {State: rc.ApplyStateAcknowledged}},
+			statuses: map[string]rc.ApplyStatus{},
```

Comment:
can you explain why this one disappeared?

Reply:
It was literally wrong: the way the test was written made it work even though it should not have